ui/storybook/README.md

```diff
@@ -0,0 +1,122 @@
+# Storybook
+
+This package is used to generate documentation for Perses UI components using [Storybook](https://storybook.js.org/).
+
+> This project currently uses the [v7 beta](https://storybook.js.org/docs/7.0/react/get-started/introduction) of Storybook. Be sure to reference the associated [v7 beta documentation](https://storybook.js.org/docs/7.0/react/get-started/introduction).
+
+## Getting started
+
+### Running storybook
+
+- Run storybook in development mode: `npm run storybook`
+
+### Building storybook
+
+- Build only: `npm run storybook:build`
+- Build & serve static assets from build: `npm run storybook:serve`
+
+## Creating stories
+
+### Where should stories live?
```

Comment:
How do folks feel about this? I think this approach makes it easier to remember to add stories because they're colocated like unit tests.

Reply:
+1 I think colocated makes sense for now